contrib/zsh_autocomplete

```diff
@@ -0,0 +1,14 @@
+# zsh programmable completion for gctcli
+# source zsh_autocomplete
+
+_gctcli() {
+
+  local -a opts
+  opts=("${(@f)$(_CLI_ZSH_AUTOCOMPLETE_HACK=1 ${words[@]:0:#words[@]-1} --generate-bash-completion)}")
+
+  _describe 'values' opts
+
+  return
+}
+
+compdef _gctcli gctcli
```

Comment:
This file will need to have its line encoding changed from CRLF to LF or zsh will be unhappy

Before:
```sh
➜ source zsh_autocomplete
zsh_autocomplete:3: command not found: ^M
zsh_autocomplete:11: parse error near `gctcli'
```

After:
```sh
➜ source zsh_autocomplete

gocryptotrader/contrib ( engine[!]) 
➜ gctcli 
addevent                           enablesubsystem                    getexchangeotp                     getorder                           getticker                          submitorder                      
addportfolioaddress                getaccountinfo                     getexchangeotps                    getorderbook                       gettickers                         whalebomb                        
cancelallorders                    getcommsrelayers                   getexchangepairs                   getorderbooks                      gettickerstream                    withdrawcryptocurrencyfunds      
cancelorder                        getconfig                          getexchanges                       getorderbookstream                 h                                  withdrawfiatfunds                
disableexchange                    getcryptocurrencydepositaddress    getexchangetickerstream            getorders                          help                                                                
disableexchangepair                getcryptocurrencydepositaddresses  getforexproviders                  getportfolio                       removeevent                                                         
disablesubsystem                   getevents                          getforexrates                      getportfoliosummary                removeportfolioaddress                                              
enableexchange                     getexchangeinfo                    getinfo                            getrpcendpoints                    setloggerdetails                                                    
enableexchangepair                 getexchangeorderbookstream         getloggerdetails                   getsubsystems                      simulateorder                          
```